Squablr/Assets.xcassets/AppIcon-2.appiconset/Contents.json

```diff
@@ -0,0 +1,98 @@
+{
```

Comment:
This is fine this time but again, you need to make sure all your PR's are doing one specific thing. Adding icons should have been a separate PR.

Reply:
Ok, will do.